frontend/fingertips-frontend/app/chart/page.test.tsx

```diff
@@ -55,6 +56,12 @@ describe('Chart Page', () => {
       areaCodes: ['A001', areaCodeForEngland],
       indicatorId: indicatorIdForPopulation,
     });
+    expect(
+      mockIndicatorsApi.getHealthDataForAnIndicator
+    ).toHaveBeenNthCalledWith(3, {
+      areaCodes: [areaCodeForEngland],
+      indicatorId: 1,
+    });
```

Comment:
I think this call should be part of the first call to get the healthData for the indicator. The endpoint takes an array of areaCodes so we include the areaCodeForEngland as part of that array. 

This way we don't have to make an additional call